--- adns-1.0/configure.in~	Mon Nov 19 02:41:12 2001
+++ adns-1.0/configure.in	Mon Nov 19 02:41:12 2001
@@ -25,7 +25,7 @@
 AC_CONFIG_HEADER(src/config.h)
 
 dnl DPKG_CACHED_TRY_COMPILE(<description>,<cachevar>,<include>,<program>,<ifyes>,<ifno>)
-define(DPKG_CACHED_TRY_COMPILE,[
+define([DPKG_CACHED_TRY_COMPILE],[
  AC_MSG_CHECKING($1)
  AC_CACHE_VAL($2,[
   AC_TRY_COMPILE([$3],[$4],[$2=yes],[$2=no])
